### PR TITLE
Inventory Viewer: Fix stackable items not showing stack of "1"

### PR DESCRIPTION
**Pull request by Puskunalis**
Before:
image

After:
image

### CODE REVIEW

File: runelite-client/src/main/java/net/runelite/client/plugins/inventoryviewer/InventoryViewerOverlay.java [hunk 1]
```diff
@@ -101,6 +98,15 @@ public Dimension render(Graphics2D graphics)
 
 	private BufferedImage getImage(Item item)
 	{
-		return itemManager.getImage(item.getId(), item.getQuantity(), item.getQuantity() > 1);
+		ItemComposition itemComposition = itemManager.getItemComposition(item.getId());
+		
+		if (itemComposition.isStackable())
```
Comment (Kamielvf) [inline runelite-client/src/main/java/net/runelite/client/plugins/inventoryviewer/InventoryViewerOverlay.java]: This whole if/else block is `return itemManager.getImage(item.getId(), item.getQuantity(), itemComposition.isStackable());` ?

Comment (Puskunalis) [reply runelite-client/src/main/java/net/runelite/client/plugins/inventoryviewer/InventoryViewerOverlay.java]: Didn't realize I could just do that. Thanks for letting me know!

File: runelite-client/src/main/java/net/runelite/client/plugins/inventoryviewer/InventoryViewerOverlay.java [hunk 2]
```diff
@@ -29,10 +29,7 @@
 import java.awt.Point;
 import java.awt.image.BufferedImage;
 import javax.inject.Inject;
-import net.runelite.api.Client;
-import net.runelite.api.InventoryID;
-import net.runelite.api.Item;
-import net.runelite.api.ItemContainer;
+import net.runelite.api.*;
```
Comment (Nightfirecat) [inline runelite-client/src/main/java/net/runelite/client/plugins/inventoryviewer/InventoryViewerOverlay.java]: Don't use star imports. Check the wiki for our code conventions on this.

File: runelite-client/src/main/java/net/runelite/client/plugins/inventoryviewer/InventoryViewerOverlay.java [hunk 3]
```diff
@@ -101,6 +98,8 @@ public Dimension render(Graphics2D graphics)
 
 	private BufferedImage getImage(Item item)
 	{
-		return itemManager.getImage(item.getId(), item.getQuantity(), item.getQuantity() > 1);
+		ItemComposition itemComposition = itemManager.getItemComposition(item.getId());
+
```
Comment (Nightfirecat) [inline runelite-client/src/main/java/net/runelite/client/plugins/inventoryviewer/InventoryViewerOverlay.java]: Remove this empty line.
```suggestion
```

File: runelite-client/src/main/java/net/runelite/client/plugins/inventoryviewer/InventoryViewerOverlay.java (hunk 2 above)
Comment (Puskunalis) [reply runelite-client/src/main/java/net/runelite/client/plugins/inventoryviewer/InventoryViewerOverlay.java]: Okay, I think IntelliJ did it automatically. I'll fix both things in a second.

**Pull request merged**